streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -96,13 +101,16 @@ public void testReprocessingFromScratchAfterReset() throws Exception {
         final KeyValue<Object, Object> result2 = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(resultTopicConsumerConfig, OUTPUT_TOPIC_2, 1).get(0);
 
         streams.close();
+        TestUtils.waitForCondition(this.consumerGroupInactive, 5 * STREAMS_CONSUMER_TIMEOUT,
```

Comment:
I was not insisting :) Just wanted to know if you want to use 10000ms or want to use 5 times the consumer timeout. Sounds good.